.env.example

```diff
@@ -157,7 +157,7 @@ GRADIENT_60 = 'rgba(42, 47, 52, 0.6)'
 GRADIENT_85 = 'rgba(42, 47, 52, 0.85)'
 
 ## physionet background image(direct download url)
-BACKGROUND_IMAGE='https://physionet.org/static/images/background.jpg'
+BACKGROUND_IMAGE='https://cdn.vox-cdn.com/thumbor/WbjC4KbwavUienc3q7DEIn1xMsg=/0x0:1440x900/1200x628/filters:focal(697x524:698x525)/cdn.vox-cdn.com/uploads/chorus_asset/file/22310830/NmJgg.jpg'
```

Comment:
If there is an existing background, we can use that. Otherwise you can find a open licensed image instead (possibly the one in your previous link?).

Reply:
or because we kind of really need to merge this PR, i can remove the commit about image upload(and for now use the physionet background from physionet.org) and add image in another PR.

Reply:
> or because we kind of really need to merge this PR, i can remove the commit about image upload(and for now use the physionet background from physionet.org) and add image in another PR.

Sounds good. Looks to me like there might be some copyright issues with Nyan Cat that it would be best to avoid: https://en.wikipedia.org/wiki/Nyan_Cat